sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/OpenCoverReportParser.java

```diff
@@ -119,6 +128,13 @@ private void handleSegmentPointTag(XmlParserHelper xmlParserHelper) {
           }
 
           coverage.addHits(identifiedFile, line, vc);
+
+          if (line == endLine) {
+            String elementIdentifier = String.format("%d-%d-%d-%d", line, endLine, startColumn, endColumn);
```

Comment:
here I use `elementIdentifier` and in the LineComplexCoverage class I use `methodName` - probably `elementIdentifier` is better, right?

Reply:
Indeed, `metodName` is confusing since it's not really a method name 😄. In the context of `OpenCover` I think that this is actually a `segmentPointRange` or `segmentPointIdentifier`.

Reply:
I think it's `sequence`